bills-app/src/main/webapp/script.js

```diff
@@ -767,26 +767,35 @@ const mockResponse = {
     }]
 }
 
-// Fetched the upcoming bills from the api
-const getUpcomingBills = () => {
+function getUpcomingBills() {
     //api key is declared in api.js as: 'proPublicaApiKey'
-    const url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
+    const url="https://api.propublica.org/congress/v1/bills/upcoming/house.json";
     fetch(url, {
         method: "GET", 
         headers: {
             "X-API-Key": proPublicaApiKey
         }
     }).then(response => response.json()).then((upcomingBills) => {
-        //printing the json file to console
-        console.log(upcomingBills);
-
-        //Prints out the contents of "status" from house.json into the website. 
-        //If everything goes well, prints out "OK"
-        document.getElementById('API-status-container').innerText = upcomingBills.status;
+        //Prints out the contents of "status" from house.json
+        //If everything goes well, prints out "OK" to console
+        console.log(upcomingBills.status);
+        upcomingBills.results[0].bills.map(i => {
```

Comment:
I think you should also call clearBillData() before you render all upcoming bills. While this won't affect anything right now, since nothing is rendered before upcoming bills, it's good practice and makes it easy to switch from searched bills back to upcoming bills if that's implemented.